src/config.ts

```diff
@@ -66,7 +66,7 @@ const validators = {
   REACT_APP_FEATURE_YAT: bool({ default: false }),
   REACT_APP_FEATURE_AXELAR: bool({ default: false }),
   REACT_APP_FEATURE_RAINBOW_CHARTS: bool({ default: false }),
-  REACT_APP_FEATURE_MTPELERIN_FIAT_RAMP: bool({ default: false }),
+  REACT_APP_FEATURE_ONRAMPER_FIAT_RAMP: bool({ default: false }),
```

Comment:
plz restore old key in addition to new one

Reply:
done